From 28c9631186bd853ba7bb87635838f7c6d78796c5 Mon Sep 17 00:00:00 2001
From: "Ethan J. Brown" <Iristyle@github>
Date: Thu, 14 Aug 2014 14:44:50 -0700
Subject: [PATCH] (PUP-3060) Remove DL is deprecated warning

 - This extraneous warning message may mislead users into believing
   that something is wrong with Puppet when it is not.  For more
   details see: http://stackoverflow.com/questions/15590450
---
 ext/dl/lib/dl.rb | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ext/dl/lib/dl.rb b/ext/dl/lib/dl.rb
index 8e615ae..76784f3 100644
--- a/ext/dl/lib/dl.rb
+++ b/ext/dl/lib/dl.rb
@@ -5,7 +5,8 @@
 rescue LoadError
 end
 
-warn "DL is deprecated, please use Fiddle"
+# http://stackoverflow.com/questions/15590450/ruby-2-0-0p0-irb-error-dl-is-deprecated-please-use-fiddle
+# warn "DL is deprecated, please use Fiddle"
 
 module DL
   # Returns true if DL is using Fiddle, the libffi wrapper.
-- 
2.0.4
